docker-build/src/main/docker/Dockerfile.native

```diff
@@ -0,0 +1,27 @@
+####
```

Comment:
Why is this file added when not used?

Reply:
Seems it's used by `NativeConfigurationIT` added after this module. So I guess this file can stay.